### PR TITLE
Do not display docker containers logs during tests execution

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -218,27 +207,18 @@ else
...
-
 # wait until hadoop processes is started
 retry check_hadoop
 stop_unnecessary_hadoop_services
 
 # start presto containers
 environment_compose up -d ${PRESTO_SERVICES}
 
-# start docker logs for presto containers
-environment_compose logs --no-color -f ${PRESTO_SERVICES} &
```
Comment (cawallin) [inline presto-product-tests/bin/run_on_docker.sh]: The Presto logs are actually quite helpful for debugging issues -- I've used them to see if Presto OOMs and what exactly was happening when it did.

**Comment by kokosing** [comment]
@cawallin Thank you for your feedback! I addressed your comment.

**Comment by arhimondr** [comment]
Displaying the container logs is especially useful when debugging various problems with container startup. Would you mind to introduce an environment variable instead of simply removing them? Or at least let the logger run during the startup and then stop the logger right before starting the tests?

**Comment by kokosing** [comment]
@arhimondr 

- I do not think that hiding this behind env variable is a good thing. That code might become wrong really soon.
- running it only for startup does change much

I decided to leave it as it is for now.

**Pull request closed without merging**